cmd/influx/user.go

```diff
@@ -187,35 +204,46 @@ func (b *cmdUserBuilder) cmdCreateRunEFn(*cobra.Command, []string) error {
 		return err
 	}
 
-	orgID, err := b.org.getID(dep.orgSvc)
-	if err != nil {
-		return err
+	var setPassErr error
+	if b.password != "" {
+		setPassErr = dep.passSVC.SetPassword(ctx, user.ID, b.password)
+	} else {
+		log.Warn("No initial password set on user, use `influx user password`", zap.String("user", b.name))
 	}
 
-	pass := b.password
-	if orgID == 0 && pass == "" {
-		return b.printUser(userPrintOpts{user: user})
+	orgID, setOrgErr := b.org.getID(dep.orgSvc)
```

Comment:
Do we want to bail earlier (i.e., here) if setPassErr != nil, and not try to set the Org?

Reply:
I'll restore the previous bail-out order (org first if ID is specified, then password). Better to not rock the boat too much on a patch release.